doc/en-us/dev/js/event_listener.md

```diff
@@ -1,54 +1,54 @@
-玩家离开、玩家发送消息等等。种种的行为或动作都对应一个事件对象作为它们的抽象，事件对象中带有这个行为或者动作中的一些
-关键信息，比如玩家发送消息事件对象中就带有发送事件的玩家对象、发送的消息字符串等等，而玩家对象又包含这个玩家的
-位置、生命值、物品栏等等数据。  
+An event object is generated whenever a specific action occurs in the server. An event is a description of some behavior or action, such as a player logging in, a
+player leaves, player sends a message, and so on. Each kind of behavior or action has an event object as its abstraction, and the event object carries some key information about the behavior or action.
+The event object contains some key information about the behavior or action, e.g., the player object that sent the event, the message string sent, etc., and the player object contains the player's
+The player object contains the player's location, life value, item bar, etc.
 
-插件可以向PNX注册**监听器**来在事件发生的时候做出反应。每个事件对象都包含了对事件的描述，通常事件监听器会在事件发生之前被调用，
-此时监听器可以通过取消事件来阻止事件真的发生，如果没有任何事件监听器取消了这个事件，那么它就会真正地在游戏内发生。
-监听器的执行是单线程的，所有监听器在注册的时候都要给PNX提供它的优先级，PNX会按照优先级将监听器排序，然后依次执行监听器。
-很可能在你的插件的监听器被执行之间这个事件已经被取消了，但通常这不会影响你的监听器的执行，你的监听器仍然会被执行，你可以通过
-`event.isCancelled()`来判断事件是否被取消了（后面会详细讲解）。
-**特别注意的是，优先级越高，执行顺序越往后排。也就是说，优先级越低，越优先执行！**   
-~~*你问为啥不是优先级越高越优先？我也这么认为，但是NKX就是这样奇怪地设计的，为了兼容NK生态我们不得不做出这个反人类的妥协*~~
+Plugins can register **listeners** with PNX to react to events as they occur. Each event object contains a description of the event, and usually the event listener is called before the event occurs.
+At this point the listener can prevent the event from actually happening by cancelling it, if no event listener cancels the event, then it will actually happen within the game.
+The execution of listeners is single-threaded, all listeners are given its priority to PNX when they are registered, and PNX will sort the listeners by priority and then execute the listeners in order.
+It is likely that this event will have been cancelled between the time your plugin's listener is executed, but this usually does not affect the execution of your listener, which will still be executed, and you can get it to work with the
+`event.isCancelled()` to determine if the event has been cancelled (more on this later).
+**Special note is that the higher the priority, the further back in the execution order you go. That is, the lower the priority, the higher the execution priority! **   
+~~*Why not the higher the priority, the higher the priority, you ask? I think so too, but that's how NKX is strangely designed, and we had to make this anti-human compromise to be compatible with the NK ecosystem *~~.
 
-监听器参与事件的过程如下面的流程如所示：  
+The listener participates in the event process as shown in the following flow:
 
 ![PNX事件流程图](../../../image/js_tutorial/PNX事件流程图.svg)  
 
-PNX中提供了非常多事件，您可以在 [事件对照表](../../res/事件对照表.html) 中查看。表中每个事件都标注了类名，
-您可以在插件中使用这个类名来监听这个事件，点击类名也可以进入对应的JavaDoc查看更多细节。  
+PNX provides a very large number of events that you can use in the [Event Comparison Table](../../res/事件对照表.html) in the table. Each event in the table is labeled with the name of the category of the event.
```

Comment:
It should be `[Event List](../res/event_list.html)`

Reply:
fix next commit

Reply:
fix in commit 11790750ed1420b783524860fab09c4dcbf3704d